internal/saucecloud/espresso.go

```diff
@@ -52,15 +52,24 @@ func (r *EspressoRunner) RunProject() (int, error) {
 		return exitCode, err
 	}
 
-	passed := r.runSuites(appFileID, testAppFileID)
+	var otherAppsIDs []string
+	for _, o := range r.Project.Espresso.OtherApps {
+		ID, err := r.uploadProject(o, otherAppsUpload)
+		if err != nil {
+			return exitCode, err
+		}
+		otherAppsIDs = append(otherAppsIDs, fmt.Sprintf("storage:%s", ID))
```

Comment:
Hm. I see why you are doing it, but this is introducing a really sneaky inconsistency.
`appFileID` and `testAppFileID` are both actual storage IDs (without the `storage:` prefix).
The nomenclature of `otherAppsIDs` suggests the same type of content, but you are prefixing `storage:`, therefore breaking the pattern ⚠️

Reply:
Maybe I need to discuss with MOBA team about this pattern and see if they can do some unification for these params.